ofrak_core/test_ofrak/components/test_pe_component.py

```diff
@@ -7,8 +7,13 @@
 from ofrak.core.pe.model import Pe, PeSection, PeSectionFlag, PeSectionHeader, PeOptionalHeader
 
 
-@pytest.fixture(params=["jumpnbump.exe", "kernel32.dll"])
-async def pe_root_resource(ofrak_context: OFRAKContext, request):
+@pytest.fixture(params=["jumpnbump.exe", "kernel32.dll", "x64kernel32.dll"])
+async def pe_root_resource(ofrak_context: OFRAKContext, request) -> Resource:
+    return await _get_test_resource_from_file_name(ofrak_context, request.param)
+
+
+@pytest.fixture(params=["kernel32.dll", "x64kernel32.dll"])
+async def kernel32_dll(ofrak_context: OFRAKContext, request) -> Resource:
```

Comment:
```suggestion
async def kernel_dll(ofrak_context: OFRAKContext, request) -> Resource:
```

Reply:
If applied would need to fix up usages of fixture

Reply:
64bit kernels in windows are still kalled `kernel32.dll` i think this is fine. Hence why i named the new test file `x64_kernel32.dll`.